testsuite/features/step_definitions/api_common.rb

```diff
@@ -234,7 +234,8 @@
   id = description = "#{client}_key"
   base_channel = LABEL_BY_BASE_CHANNEL[BASE_CHANNEL_BY_CLIENT[client]]
   key = $api_test.activationkey.create(id, description, base_channel, 100)
-  raise if key.nil?
+  raise 'Error creating activation key via the API' if key.nil?
```

Comment:
I thought we were trying to do 
```suggestion
  raise StandardError, 'Error creating activation key via the API' if key.nil?
```
instead?